### ISSUE TITLE
Didn't get data after sign in through yarn start:alternativeAuth

### ISSUE THREAD

**Issue by AlokTakshak** [mozilla/bugzilla-dashboard#107]
started application using yarn start:alternativeAuth
then signed in using my google account ,shows me signed in but not getting any data, showing the attached error. 

![image](https://user-images.githubusercontent.com/26347838/57427014-a5b32d00-723f-11e9-9b9e-ff7935f64b9e.png)

**Comment by AlokTakshak** [comment]
@sylvestre if you could just help me with this sign in issue so that i can start working on some other issues
thanks

**Comment by sylvestre** [comment]
@AlokTakshak did you make progress on this?
Thanks

**Comment by AlokTakshak** [comment]
@sylvestre No, even if i make the changes in components ,still won't know how it looks

**Comment by sylvestre** [comment]
You should get something like: 
![image](https://user-images.githubusercontent.com/733326/58026067-00893600-7b16-11e9-92cb-f8487c0ed6ab.png)

for the reportees

**Comment by AlokTakshak** [comment]
> started application using yarn start:alternativeAuth
> then signed in using my google account ,shows me signed in but not getting any data, showing the attached error.
> 
> ![image](https://user-images.githubusercontent.com/26347838/57427014-a5b32d00-723f-11e9-9b9e-ff7935f64b9e.png)
> `secretsClient.get(config.taskclusterSecrets.orgData);` in getAllReportees
> is returning a rejected promise.

@sylvestre still getting the same error

**Comment by sylvestre** [comment]
Sure, we haven't made any change to fix it.
Did you try with https://github.com/mozilla/bugzilla-dashboard/pull/80 ?

**Comment by AlokTakshak** [comment]
@sylvestre `getAllFakeReportees.js` doesn't exist in the repo
